doc/ci.md

```diff
@@ -20,8 +22,12 @@ Steps for all PRs are:
 * call `./ci/build-scripts/build-backend.sh`, which builds the rust binary as well as the bindings' definition via `typeshare`
 * call `./ci/build-scripts/build-frontend.sh`, which does a full release build of the frontend
 
-It will be available for public access on [pr.permaplant.net](pr.permaplant.net).
+## Deploy to PR Environment
+
+Each and every pull request will be deployed on a publicly available instance on [pr.permaplant.net](pr.permaplant.net).
+
+Since there is only one PR Environment available, the last built PR wins.
```

Comment:
If you explained what an agent is before I suggest (otherwise use instance):

```suggestion
Since there is only one agent for PRs available, the last built PR wins.
```